src/test/resources/pages/studentHomeFeedbackDeletedHTML.html

```diff
@@ -24,7 +24,7 @@ <h1>
         </a>
       </span>
     </div>
-    <table class="table-responsive table table-striped">
+    <table class="table-responsive table table-striped table-bordered margin-0">
```

Comment:
A lot of `table class` lines have `table-bordered magin-0` inserted. What is this for?

Reply:
It draws lines to differentiate columns of a table.